community/teams/cuda.tt

```diff
@@ -0,0 +1,61 @@
+[% WRAPPER layout.tt title="CUDA Team" handlesLayout=1 %]
+
+<div class="page-title">
+  <div>
+    <a href="[% root%]community/index.html#governance-teams">Community</a>
+    <span>→</span>
+  </div>
+  <h1>CUDA team</h1>
+</div>
+
+<section class="governance-description">
+  The CUDA team is responsible for all things CUDA-related in <a href="https://nixpkgs.org">Nixpkgs</a>.
```

Comment:
```suggestion
  The CUDA team is responsible for everything related to <a href="https://developer.nvidia.com/cuda-zone">CUDA</a> in <a href="https://github.com/NixOS/nixpkgs">Nixpkgs</a>.
```

We should not link to the same page we're on. Nixpkgs is primarily a GitHub repo. Also it probably makes sense to link to what CUDA is, as we can't expect everyone to know what that is. This page is for the general audience.

Reply:
> Not a goal

Looking at https://nixos.org/community/teams/nix.html as an example, we could put this in "responsibilities"

Reply:
Fixed in newest